config/default/manager_auth_proxy_patch.yaml

```diff
@@ -51,8 +51,13 @@ spec:
         args:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
+        - "--pprof-bind-address=:8082"
         - "--leader-elect"
         - "--wds-name={{.Values.ControlPlaneName}}"
         - "--its-name={{.Values.ITSName}}"
         - "--api-groups={{.Values.APIGroups}}"
         - -v={{.Values.ControllerManager.Verbosity}}
+        ports:
+        - containerPort: 8082
+          protocol: TCP
+          name: debug-pprof
```

Comment:
I think that it is best if we leave most of the configuration in the base YAML file rather than in this delta. The base file is in `config/manager/manager.yaml`.

Reply:
The separation of stuff in the files is maddening to me.
I put it in this file because the other containerport was in this file and the flag that open the port is in this file.
I'm ok to move it if you prefer

Reply:
I agree. The use of kustomize here brings us only pain.

Reply:
However, I think that the code generation tool that we use produces output that requires modification.

For now, let's just minimize what goes in the delta. I think that this port declaration would go better in the base YAML file.

Reply:
ok, done